app/assets/javascripts/static.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Application entry point.
+ */
+require([
+  'jquery',
+  'underscore',
+  'Class',
+  'backbone',
+  'views/AboutView'
+], function($, _, Class, Backbone, AboutView) {
+
+  'use strict';
+
+  var Static = Class.extend({
+
+    $el: $('body'),
+
+    init: function() {
+      this._initViews();
+    },
+
+    /**
+     * Initialize the map by starting the history.
+     */
+    _initApp: function() {
+      if (!Backbone.History.started) {
+        Backbone.history.start({pushState: true});
+      }
+    },
+
+    /**
+     * Initialize Application Views.
+     * CAUTION: Don't change the order of initanciations if
+     * you are not completely sure.
+     */
+    _initViews: function() {
+      new AboutView();
```

Comment:
Ok ! for me doesn't make sense create a view only for a event and also we need a view for the page because we forgot initialize the`SidebarNavView` and it and makes the AboutView necessary so I will add the commit and feel free to merge 😄